src/Equinor.ProCoSys.IPO.Domain/AggregateModels/InvitationAggregate/Invitation.cs

```diff
@@ -251,6 +251,27 @@ public void CompleteIpo(Participant participant, string participantRowVersion, P
             AddPreSaveDomainEvent(new IpoCompletedEvent(Plant, ObjectGuid));
         }
 
+        public void UnCompleteIpo(Participant participant, string participantRowVersion)
```

Comment:
Add tests for this in InvitationTests
I see that it is not done for unaccept either, could you please do it for unaccept as well?